docs/dcp2_system_design.rst

```diff
@@ -0,0 +1,1465 @@
+   generated using the same namespace UUID as for
+   `Contributor-generated matrices (CGMs)`_ and the name
+   ``dcp2``.
+
+-  The ``analysis_file.file_core.content_description`` field of both
+   intermediate and project-level matrices is set to::
+
+       {
+           "text": "DCP/2-generated matrix",
+           "ontology": "data:3917",
+           "ontology_label": "Count Matrix"
+       }
+
+-  The ``analysis_file`` entities are linked to inputs via
+       ``analysis_process`` entities.
+
+-  Each ``analysis_process`` is described by an ``analysis_protocol``
+
+-  The ``analysis_protocol.protocol_core.protocol_id`` will be
+       ``optimus_post_processing_v1.0.0`` for stratified, project-level
+       matrices and ``optimus_v1.0.0`` for the intermediate ones
+
+Any intermediate matrices created during the processing are described as
+``analysis_file``, but the ``analysis_file.provenance.submitter_id``
+property is omitted.
+
+.. figure:: media/dcp2-matrices.png
+
+   Graphical interpretation of how DCP/2 generated matrices are
+   linked to the experimental graph
+
+DCP/1-generated matrices
+------------------------
+
+-  The ``supplementary_file.provenance.submitter_id`` field is set to
+       ``c9efbb15-c50c-5796-8d15-35e9e1219dc5``. This UUIDv5 was
+       generated using the same namespace UUID as for
+       `Contributor-generated matrices (CGMs)`_ and the name
+       ``dcp1 matrix service``.
+
+WIP: Contributor-generated matrices, organically described in metadata
+----------------------------------------------------------------------
+
+Since these matrices are described organically using the metadata graph,
+they more closely resemble DCP/2 matrices instead of the CGMs described
+in section ....
+
+So Ingest would describe these matrices using an appropriate metadata
+graph, similar to how analysis describes their DCP/2-generated matrices.
+
+This would not require any schema changes but the hacky submitter_id
+mechanism would have to be used.
+
+TBD: How do we distinguish these organically described CGMs from DCP/2
+generated matrices? [Probably via a dedicated submitter ID)
+
+TBD: Do we migrate the current non-organically described CGMs to this
+new scheme? If so, who does it and how does this relate to versioning?
+
+TBD: What about intermediate files like BAMs or intermediate matrices?
+Should they be required or can the graph skip them and instead use a
+simple process to connect the primary sequence data (fastq for example)
+to the project-level matrices?
+
+WIP: Data Browser & Azul catalogs
+---------------------------------
+
+Migrated from in `another document`_.
+
+.. _another document: https://docs.google.com/document/d/1ok7VpGJJaPP-jqP_9pU3A5KsMdPmRDBXS-KfsNu5-Nc/edit?usp=sharing
+
+We change the Data Browser URL scheme to include an optional initial
+version identifier in the URL path
+
+.. parsed-literal::
+
+    https://data.humancellatlas.org/explore/\ **v2/**\ projects?catalog=dcp1&filter=...
```

Comment:
(nitpick) This displays with `https://data.humancellatlas.org/explore/` as a link to `https://data.humancellatlas.org/explore/`. Is there any way to prevent the auto-linking?

Reply:
Not that I've found :/